stackslib/src/chainstate/stacks/db/headers.rs

```diff
@@ -154,7 +153,7 @@ impl StacksChainState {
             anchored_block_cost,
             block_size_str,
             parent_id,
-            u64_to_sql(affirmation_weight)?,
+            u64_to_sql(0)?, // TODO: remove this
```

Comment:
Hmm. This makes me wonder if the safer thing to do with this PR is to actually drop the column. That way, we can be sure that no SQL queries actually use the affirmation weight column anymore (because they'll error, which I think is safer than returning bad data).